### PR TITLE
Completed

### PR DESCRIPTION
**Pull request by Siri-isha**

### CODE REVIEW

File: LongestSubstring.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import java.util.HashSet;
+import java.util.Set;
+//TC - O(n)
+// SC - O(N) - N is length of resultant longest substring
```
Comment (ishika2741) [inline LongestSubstring.java]: Resultant space does not count in complexity

**Review by ishika2741 — COMMENTED** [review]
Good work!

**Comment by ishika2741** [comment]
Good work!

File: LongestSubstring.java (hunk 1 above)
Comment (Siri-isha) [reply LongestSubstring.java]: Yes, modified the SC to O(1)

**Pull request closed without merging**